src/main/java/mil/dds/anet/resources/PositionResource.java

```diff
@@ -164,8 +165,25 @@ public Integer updatePosition(@GraphQLRootContext Map<String, Object> context,
     return numRows;
   }
 
+  @GraphQLMutation(name = "updatePositionHistory")
+  public int updatePositionPreviousPeople(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "position") Position pos) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    final Position existing = dao.getByUuid(pos.getUuid());
+    assertCanUpdatePosition(user, existing);
+    if (AnetObjectEngine.getInstance().getPositionDao().hasPersonConflict(pos)) {
+      throw new WebApplicationException(
+          "There are time conflict between time you selected and selected positions's history ",
```

Comment:
Fix typo's, and the message is also not clear to me. What are the conflicts? I assume some of the history entries on the position passed as input, conflict with the position history of the person in those entries?